test/benchmarks.ts

```diff
@@ -151,3 +159,21 @@ async function getBenchmark(repoInfo: RepoInfo) {
     },
   });
 })();
+
+/*
```

Comment:
These seem like module level comments. Do you think they should be moved to the top of the module for better visibility?

Reply:
unintentional